CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contribution guide
+
+Hey there, and thank you for helping out! This guide will help you set up your development environment, and submit your pull request to the oVirt Terraform Provider.
+
+## Setting up a development environment
+
+Terraform providers are written in the [Go programming language](https://golang.org/), so make sure you set up a local Go SDK. At the time of writing you will need Go version `1.14`, but make sure to check the [go.mod](go.mod) file for the current version.
+
+Other than that, you will need a working oVirt engine with at least one host and a working storage domain. The test suite requires roughly 2 GB of disk space.
+
+## Developing your patch
+
+When developing your patch you will need to keep a few things in mind. In no particular order, these are the following:
+
+### One change = one PR
+
+Please make sure that your patch contains as little as feasible. Having a smaller change makes it easier to test and review.
+
+Formatting changes, adding or removing empty lines should be avoided under all circumstances because they unnecessarily bloat the patch size. If you want to submit a formatting change please do so in a separate PR.
+
+### Use and write tests
+
+Tests are an integral part of writing a Terraform provider as manual testing every change is nearly impossible.
+
+You can run the oVirt provider tests using the following command:
+
+```bash
+OVIRT_URL=https://your-ovirt-engine-host/ovirt-engine/api/;OVIRT_USERNAME=admin@internal;OVIRT_PASSWORD=your-ovirt-password;TF_ACC=1;OVIRT_INSECURE=1 go test -v ./...
```

Comment:
i couldnt run it with this command , ended up using : `IRT_URL=https://your-ovirt-engine-host/ovirt-engine/api/ OVIRT_USERNAME=admin@internal OVIRT_PASSWORD=your-ovirt-password TF_ACC=1 OVIRT_INSECURE=1 go test -v ./...`